app/assets/javascripts/discourse/app/initializers/click-interceptor.js

```diff
@@ -3,8 +3,9 @@ import interceptClick from "discourse/lib/intercept-click";
 
 export default {
   name: "click-interceptor",
-  initialize() {
-    $("#main").on("click.discourse", "a", interceptClick);
+  initialize(container, app) {
+    this.selector = app.rootElement;
+    $(this.selector).on("click.discourse", "a", interceptClick);
```

Comment:
can we avoid jquery while we are at it please?

Reply:
Not easily - this is using JQuery's 'selector' filter to receive events from all `<a>` tags within the rootElement. To move away from JQuery, we'd have to re-implement that feature functionality. 

We should totally do it, but I think it should be a separate PR (and should probably be merged after the next release)